tutorial/under-the-hood/Compilation.md

```diff
@@ -342,19 +342,14 @@ scenario. In XSpec, this is either an XSLT template (named or
 rule) or an XPath function (written either in XSLT or XQuery).
 Here, we use it to refer to the three ways to refer to the SUT
```

Comment:
"three" can become "two".

Also, can you delete the phrase "`x:apply` (not implemented yet), " on line 495? (GitHub doesn't let me put a comment directly on that line.)

Reply:
Done in e50b6f92c8cc919b7d0467e7677cca9bfed123ef.